collada/schema.py

```diff
@@ -435,7 +435,7 @@
     <xs:simpleType name="URIFragmentType">
         <xs:annotation>
             <xs:documentation>
-            This type is used for URI reference which can only reference a resource declared within it's same document.
+            This type is used for URI reference which can only reference a resource declared within its same document.
```

Comment:
I think we should probably leave this file alone, because it's a copy of the official schema from the collada spec:
https://www.khronos.org/files/collada_schema_1_4

Reply:
I'll remove this commit from the PR. I'll try to submit these changes upstream.